python/cudf/cudf/tests/test_indexing.py

```diff
@@ -1610,9 +1610,12 @@ def test_dataframe_loc_inplace_update_with_invalid_RHS_df_columns():
     actual = gdf.loc[[0, 2], ["x", "y"]] = cudf.DataFrame(
         {"b": [10, 20], "y": [30, 40]}, index=cudf.Index([0, 2])
     )
-    expected = pdf.loc[[0, 2], ["x", "y"]] = pd.DataFrame(
-        {"b": [10, 20], "y": [30, 40]}, index=pd.Index([0, 2])
-    )
+    with pytest.warns(FutureWarning):
+        # Seems to be a false warning from pandas,
```

Comment:
Just curious what warning is pandas throwing?

Reply:
Somewhere in the code flow, pandas is trying to set `nan` to `int` column thus casting to `float` column. However, I feel this was a false warning because this was later being discarded and not part of the end result.

Reply:
I see I opened up https://github.com/pandas-dev/pandas/issues/56714 to see what the true behavior should be as if left as-is this will break in pandas 3.0 probably.